Tests/test_program/test_instr.c

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
let's add some simple unit tests:
1. with a switch statement
2. a loop where the counter is given as input
3. a function call within the same file